apps/ensindexer/src/config/config.schema.ts

```diff
@@ -131,13 +131,17 @@ const PortSchema = z.coerce
 
 const EnsRainbowEndpointUrlSchema = makeUrlSchema("ENSRAINBOW_URL");
 
-const RpcConfigsSchema = z.record(
-  z.string().transform(Number).pipe(chainIdSchema),
-  RpcConfigSchema,
-  {
+const RpcConfigsSchema = z
+  .record(z.string().transform(Number).pipe(chainIdSchema), RpcConfigSchema, {
     error: "Chains configuration must be an object mapping valid chain IDs to their configs.",
-  },
-);
+  })
+  .refine((configs) => Object.keys(configs).length > 0, {
+    message:
+      "No RPC configurations found. You must configure RPC endpoints for the chains required by your enabled plugins. See .env.local.example for guidance. Public (rate limited) RPC endpoints will not provide acceptable performance - use only private (paid) endpoints.",
+  })
+  .refine((configs) => !!configs[1], {
```

Comment:
Is RPC_URL_1 always required? If yes then this check should be here or in a invariant function?